apiserver/facades/client/application/service.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2024 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
```

Comment:
Please rename this file to interface.go to be consistent within juju.

Reply:
`service.go` is the convention we've been using in every other facade for a file containing all the service interfaces used by the facade.

Reply:
This, if this has changed since the Dqlite work started please have it circulated.

Reply:
this is why we need well-documented conventions, not just word-of-mouth